submissions/UA-girl/task_js_dom/css/styles.css

```diff
@@ -0,0 +1,273 @@
+/*Common*/
+
+body {
+    font-family: 'Josefin Sans', sans-serif;
+    font-size: 16px;
+    font-weight: 300;
+    color: #333333;
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    background-color: #f1f1f1;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+.container {
+    box-sizing: border-box;
+}
+
+@media screen and (min-width: 860px) {
+    .container {
+        width: 90%;
+        margin: 0 auto;
+        box-shadow: 0 0 30px rgba(0, 0, 0, 0.4);
+    }
+}
+
+/*Header*/
```

Comment:
I think it's a redundant comment here. To my mind, it's clear for other devs that you style the header here.